### ISSUE TITLE
Review and refactor e2e tests

### ISSUE THREAD

**Issue by slaweet** [LiskHQ/lisk-nano#973]
Lisk Nano has too many e2e tests. The number should be reduced through several strategies:
- Sometimes two e2e tests can be merged, e.g.
https://github.com/LiskHQ/lisk-nano/blob/34978fd1376ae01008fbde4e4fe01951b3f93bcb/features/voting.feature#L1-L12
- Some are already covered by unit tests, thus can be removed, e.g:
https://github.com/LiskHQ/lisk-nano/blob/34978fd1376ae01008fbde4e4fe01951b3f93bcb/features/send.feature#L18-L23
- Some should be converted to integration tests. For the sake of this issue, I would just mark them with @integration tag and there will be another issue to do the job after #733 is done. E.g.:
https://github.com/LiskHQ/lisk-nano/blob/34978fd1376ae01008fbde4e4fe01951b3f93bcb/features/menu.feature#L84-L88

Appart from that, [features/menu.feature](https://github.com/LiskHQ/lisk-nano/blob/34978fd1376ae01008fbde4e4fe01951b3f93bcb/features/menu.feature) is too big and not really just one feature. It should be split into multiple files: `registerDelegat.feature`, `registerSecondPassphrase.feature` `signMessage.feature`, `verifyMessage.feature`
